ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/acm-common-ranGen.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: common-latest
+placementBindingDefaults:
+  name: common-placement-binding
+policyDefaults:
+  # categories: []
+  #controls:
+  #  - PR.DS-1 Data-at-rest
+  namespace: ztp-common
+  # Use an existing placement rule so that placement bindings can be consolidated
+  placement:
+    labelSelector:
+      common: "true"
+      du-profile: "latest"
+  remediationAction: inform
+  severity: low
+  # standards: []
+  namespaceSelector:
+    exclude:
+      - kube-*
+    include:
+      - '*'
+  evaluationInterval:
+    compliant: 10m
+    noncompliant: 10s
+policies:
+  # Create operators policies that will be installed in all clusters
+- name: common-subscriptions-policy-latest
+  policyAnnotations:
+    ran.openshift.io/ztp-deploy-wave: "2"
+  manifests:
+    - path: source-crs/SriovSubscriptionNS.yaml
+    - path: source-crs/PtpSubscriptionNS.yaml
+    - path: source-crs/ClusterLogNS.yaml
+    - path: source-crs/StorageNS.yaml
```

Comment:
Minor nit: I think it is better to keep the NS and operatorstatus with the subscription. Customers who want to opt out of an operator will need to comment out the ns, subscription, opergroup, and status which is easier to do if they are together. I like the re-ordering though to match dependency

Reply:
good point, done.